packages/ai-chat/src/browser/ai-chat-preferences.ts

```diff
@@ -0,0 +1,31 @@
+// *****************************************************************************
+// Copyright (C) 2024 EclipseSource GmbH.
+//
+// This program and the accompanying materials are made available under the
+// terms of the Eclipse Public License v. 2.0 which is available at
+// http://www.eclipse.org/legal/epl-2.0.
+//
+// This Source Code may also be made available under the following Secondary
+// Licenses when the conditions for such availability set forth in the Eclipse
+// Public License v. 2.0 are satisfied: GNU General Public License, version 2
+// with the GNU Classpath Exception which is available at
+// https://www.gnu.org/software/classpath/license.html.
+//
+// SPDX-License-Identifier: EPL-2.0 OR GPL-2.0-only WITH Classpath-exception-2.0
+// *****************************************************************************
+
+import { AI_CORE_PREFERENCES_TITLE } from '@theia/ai-core/lib/browser/ai-core-preferences';
+import { PreferenceSchema } from '@theia/core/lib/browser/preferences/preference-contribution';
+
+export const DEFAULT_CHAT_AGENT_PREF = 'ai-features.AIChat.defaultChatAgent';
```

Comment:
In all other preferences we used lower cases and dashes, so I would do that here too. Also I would drop the additional `AI` from `AIChat`.
```suggestion
export const DEFAULT_CHAT_AGENT_PREF = 'ai-features.chat.default-chat-agent';
```

Reply:
Right, we should keep it consistent. However, I think the properties look a bit ugly with the dashes and we loose the default capitalization that Theia applies. Anyway, this should probably be a separate change that fixes it for all preferences.